src/sagas/ui.js

```diff
@@ -37,35 +35,28 @@ function* githubExport(
   }
 }
 
-export function* exportGist() {
-  yield* githubExport(
-    'GIST_EXPORTED',
-    'GIST_EXPORT_ERROR',
-    gistExportNotDisplayed,
-    gistExportDisplayed,
-  );
-}
 
 export function* popOutProject() {
   const project = yield select(getCurrentProject);
   const {source} = yield call(generatePreview, project);
   yield call(openWindowWithContent, source);
 }
 
-export function* exportRepo() {
-  yield* githubExport(
-    'REPO_EXPORTED',
-    'REPO_EXPORT_ERROR',
-    repoExportNotDisplayed,
-    repoExportDisplayed,
+
+export function* exportProject() {
+  yield* projectExport(
+    'PROJECT_EXPORTED',
+    'PROJECT_EXPORT_ERROR',
+    projectExportNotDisplayed,
+    projectExportDisplayed,
   );
 }
 
+
```

Comment:
No need for the extra line of whitespace here

Reply:
Yup!